src/rustc.rs

```diff
@@ -364,15 +395,6 @@ pub fn target_list(msg_info: &mut MessageInfo) -> Result<TargetList> {
         })
 }
 
-pub fn sysroot(msg_info: &mut MessageInfo) -> Result<PathBuf> {
-    let stdout = rustc_command()
-        .args(["--print", "sysroot"])
-        .run_and_get_stdout(msg_info)?
-        .trim()
-        .to_owned();
-    Ok(PathBuf::from(stdout))
-}
-
```

Comment:
this is not needed anymore, `rustup toolchain list -v` gives us this, for all toolchains